web/src/layouts/LoginLayout.tsx

```diff
@@ -2,7 +2,14 @@ import React, { ReactNode } from "react";
 import { Grid, makeStyles, Container, Typography, Link } from "@material-ui/core";
 import { ReactComponent as UserSvg } from "../assets/images/user.svg";
 import { grey } from "@material-ui/core/colors";
+import { useTheme, useMainColor } from '../hooks/Theme';
 
+var color = "#000";
+if (useTheme() === "dark") {
```

Comment:
A better way to retrieve the color from the theme is to use `useTheme` from `"@material-ui/core"` within the component. With that variable you can easily access the color that has been set in the material theme instead of retrieving the raw data. Thus, you would not need this snippet in every part of the app requiring the color.

Reply:
I am sorry, but i do not fully understand what you mean by this. You seem to see a better approach, which i'm glad. May i ask you to elaborate?

Reply:
There is hook called `useTheme` in the `@material-ui/core` library. Please use it instead of yours: https://material-ui.com/styles/api/#usetheme-theme